samples/snippets/src/main/java/com/example/spanner/PgSpannerSample.java

```diff
@@ -577,6 +577,86 @@ static void writeUsingDml(DatabaseClient dbClient) {
   }
   // [END spanner_postgresql_dml_getting_started_insert]
 
+  // [START spanner_postgresql_insert_dml_returning]
```

Comment:
The preferred format for new samples is to put these in separate files, and not to add them to the large monolithic sample file. See for example `PgBatchDmlSample`.

Reply:
Done.